torch/nn/modules/module.py

```diff
@@ -378,6 +380,9 @@ def to(self, *args, **kwargs):
                 the floating point parameters and buffers in this module
             tensor (torch.Tensor): Tensor whose dtype and device are the desired
                 dtype and device for all parameters and buffers in this module
+            memory_format (:class:`torch.memory_format`): the desired memory
+                format  for all parametes and buffers in this module (keyword
```

Comment:
spelling: parameters

Reply:
this doesn't seem true (I guess it depends on your definition of "desired").  But this only converts the 4-dimensional parameters, right?